### ISSUE TITLE
How to add a new rule for Quark Browser?

### ISSUE THREAD

**Issue by xiaoxuetu** [nielsbasjes/yauaa#110]
I found a problem with this useragent.
[Please update the output below to match what you expect it should be]

```

- test:
    input:
      user_agent_string: 'Mozilla/5.0 (Linux; U; Android 6.0.1; zh-CN; Nexus 6P Build/MTC20L) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.108 Quark/2.4.5.934 Mobile Safari/537.36'
    expected:
      DeviceClass  : 'Phone'
      DeviceName  : 'Google Nexus 6P'
      DeviceBrand  : 'Google'
      OperatingSystemClass  : 'Mobile'
      OperatingSystemName  : 'Android'
      OperatingSystemVersion  : '6.0.1'
      OperatingSystemNameVersion  : 'Android 6.0.1'
      OperatingSystemVersionBuild  : 'MTC20L'
      LayoutEngineClass  : 'Browser'
      LayoutEngineName  : 'Blink'
      LayoutEngineVersion  : '57.0'
      LayoutEngineVersionMajor  : '57'
      LayoutEngineNameVersion  : 'Blink 57.0'
      LayoutEngineNameVersionMajor  : 'Blink 57'
      AgentClass  : 'Browser Webview'
      AgentName  : 'Chrome Webview'
      AgentVersion  : '57.0.2987.108'
      AgentVersionMajor  : '57'
      AgentNameVersion  : 'Chrome Webview 57.0.2987.108'
      AgentNameVersionMajor  : 'Chrome Webview 57'
      AgentLanguage  : 'Chinese (S)'
      AgentLanguageCode  : 'zh-cn'
      AgentSecurity  : 'Strong security'
      WebviewAppName  : 'Mobile Safari'
      WebviewAppVersion  : '537.36'
      WebviewAppVersionMajor  : '537'
      WebviewAppNameVersionMajor  : 'Mobile Safari 537'



```

**Comment by xiaoxuetu** [comment]
The `WebviewAppName` should be `Quark` and the `AppVersion` should be `2.4.5.934`

**Comment by nielsbasjes** [comment]
Implemented
https://github.com/nielsbasjes/yauaa/blob/master/analyzer/src/main/resources/UserAgents/GoogleChrome.yaml#L1393

**Comment by nielsbasjes** [comment]
Just now I released version 5.3 which includes this.
